recipe/meta.yaml

```diff
@@ -22,7 +21,7 @@ requirements:
     - setuptools
     - wheel
   run:
-    - python >=3.6
+    - python
 
 test:
   imports:
```

Comment:
Could you remove `- python <3.10` form `test/requires`

Reply:
ok